view/src/bin/pviewd.rs

```diff
@@ -32,15 +49,31 @@ struct Opt {
     pd_port: u16,
 }
 
-#[derive(Debug, Subcommand)]
+#[derive(Debug, clap::Subcommand)]
 enum Command {
-    /// Initialize the view service with a full viewing key.
     Init {
         /// The full viewing key to initialize the view service with.
         full_viewing_key: String,
+        // The seed phrase providing spend capability
+        seed_phrase: Option<String>,
     },
     /// Start the view service.
-    Start {
+    #[clap(subcommand, display_order = 100)]
+    Start(StartCmd),
+}
+#[derive(Debug, clap::Subcommand)]
+pub enum StartCmd {
+    /// Initialize the view service without spend capability
+    View {
+        /// Bind the view service to this host.
+        #[clap(long, default_value = "127.0.0.1")]
+        host: String,
+        /// Bind the view gRPC server to this port.
+        #[clap(long, default_value = "8081")]
+        view_port: u16,
+    },
+    /// Initialize the custody service with a seed phrase
+    Custody {
```

Comment:
Rather than having subcommands on `start`, we should have the `view`/`custody` subcommands be on `init`, and have `start` run in whatever mode is specified in the config file.

Reply:
is there a use case where users will want to use a custody file containing a spend key to run the client in view mode? because if so, I figured it made sense to put the switch on `start` instead of `init`, but do we want to maybe add an explicit "start mode" field to the config file instead of just relying on the presence or absence of a spend key?

Reply:
No, I think they could do that by removing the custody config (or commenting it out)